bulk_query_gen/influxdb/influx_common.go

```diff
@@ -40,7 +42,7 @@ func (d *InfluxCommon) getHttpQuery(humanLabel, intervalStart, query string, q *
 	q.HumanDescription = []byte(fmt.Sprintf("%s: %s", humanLabel, intervalStart))
 	q.Language = d.language.String()
 
-	if d.language == InfluxQL {
+	if d.language == InfluxQL && d.version == 1 {
 		getValues := url.Values{}
 		getValues.Set("db", d.DatabaseName)
```

Comment:
Do we need to set `db` on the POST requests somehow for InfluxQL on v2?

Reply:
I think that the `else` branch of this if statement covers that -- basically the Method is declared as a post, and the url path is set further along the call chain